### PR TITLE
168 rte text color extension

### LINKED ISSUE #168: RTE Text color extension
**Is your feature request related to a problem? Please describe.**
We need the possibility to change the text color in RTE. 

Screenshot 2023-04-24 at 11 58 45

**Describe the solution you'd like**

- [ ] UI component for RTE that allows choosing a color
- [ ] Plugin component for RTE that  allows adding a specific class to the text
- [ ] Configuration that defines components with available colors and calls setting a proper class
- [ ] Configuration should contain colors defined in https://bulma.io/documentation/helpers/color-helpers/ (default, light and dark)

**Describe alternatives you've considered**
Setting just a text color instead of class.

**Additional context**
Using classes can provide some problems like:

- showing text color in the editor,
- switching between colors (changing classes)
- might not be supported by existing TipTap extensions, so we might need to implement our own
- ...

Setting just a text color might not be the best solution but it can be a first step before some more general solution.

### PR DESCRIPTION
**Pull request by martyna-ds**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [ ] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: application/extensions/bulma-color/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 Dynamic Solutions
```
Comment (paweljozwicki) [inline application/extensions/bulma-color/.eslintrc.js]: 2023 :)

File: application/backend/src/main/resources/libs/bulma/wcm/dialogs/components/richtext/configuration/.content.json
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/wcm/dialogs/components/richtext/configuration/.content.json:1]: I think this file is unnecessary :)

File: application/extensions/bulma-color/pom.xml [hunk 2]
```diff
@@ -0,0 +1,273 @@
...
+    <extensions>
+      <extension>
+        <groupId>com.github.seahen</groupId>
+        <artifactId>maven-s3-wagon</artifactId>
+        <version>1.3.3</version>
+      </extension>
+    </extensions>
+  </build>
+
+  <dependencies>
```
Comment (paweljozwicki) [inline application/extensions/bulma-color/pom.xml]: All those dependencies are not needed

File: application/extensions/bulma-color/.eslintrc.js (hunk 1 above)
Comment (martyna-ds) [reply application/extensions/bulma-color/.eslintrc.js]: Done

File: application/backend/src/main/resources/libs/bulma/wcm/dialogs/components/richtext/configuration/.content.json
Comment (martyna-ds) [reply application/backend/src/main/resources/libs/bulma/wcm/dialogs/components/richtext/configuration/.content.json:1]: Thanks, done

File: application/extensions/bulma-color/pom.xml (hunk 2 above)
Comment (martyna-ds) [reply application/extensions/bulma-color/pom.xml]: Done

**Review by paweljozwicki — APPROVED** [review]

**Pull request merged**